README.md

```diff
@@ -30,7 +29,7 @@ Create a file called `main.tf` with the following contents:
 ```hcl
```

Comment:
An approach that seems appealing is to put these examples into an examples/ subfolder (allowing for different examples):

https://github.com/dsieczko/terraform-metal-windows-starter/tree/master/examples/simple

In the root README.md we can then refer folks to those example directories or the README.md in each example directory.

Reply:
fwiw, I don't think the Terraform registry does anything special with the examples.

Reply:
But this is the Hashicorp recommendation:
https://www.terraform.io/docs/language/modules/develop/structure.html
>  If you want to include an example for how this module can be used in combination with other resources, put it in an [examples directory like this](https://github.com/hashicorp/terraform-aws-consul/tree/master/examples). Consider including a visual diagram depicting the infrastructure resources the module may create and their relationship.